src/agenda_record.cc

```diff
@@ -238,22 +246,22 @@ void write_agenda_wrapper_header(ofstream& ofs,
   // Wrapper function output parameters
   const ArrayOfIndex& ago = agr.Out();
   ofs << "        // Output\n";
-  for (ArrayOfIndex::const_iterator j = ago.begin(); j != ago.end(); j++) {
+  for (long long j : ago) {
```

Comment:
This should be `Index` or `std::int64_t`.

Reply:
Yup.  Argh, clang is messing with me :/